src/index.ts

```diff
@@ -1,34 +1,77 @@
 // Copyright (c) 2023 Cloudflare, Inc.
 // Licensed under the Apache-2.0 license found in the LICENSE file or at https://opensource.org/licenses/Apache-2.0
 
-import { TOKEN_TYPES } from './pubVerifToken.js';
-import { PrivateToken, Token } from './httpAuthScheme.js';
 import { base64url } from 'rfc4648';
-
 import { convertEncToRSASSAPSS, convertRSASSAPSSToEnc } from './util.js';
+import { BLIND_RSA } from './pub_verif_token.js';
+import { VOPRF } from './priv_verif_token.js';
+import { AuthorizationHeader, WWWAuthenticateHeader } from './auth_scheme/private_token.js';
+import { issuanceProtocolPriv, issuanceProtocolPub } from './issuance.js';
+
 export const util = { convertEncToRSASSAPSS, convertRSASSAPSSToEnc };
-export * from './pubVerifToken.js';
-export * from './httpAuthScheme.js';
+export * from './auth_scheme/private_token.js';
+export * from './pub_verif_token.js';
+export * from './priv_verif_token.js';
 export * from './issuance.js';
 
+// Privacy Pass Token Type Registry
+// Supported:
+//  - Token Type VOPRF (P-384, SHA-384)
+//  - Token Type Blind RSA (2048-bit)
+//
+// https://datatracker.ietf.org/doc/html/draft-ietf-privacypass-protocol-16#name-token-type-registry-updates
+export const TOKEN_TYPES = {
+    // Token Type Blind RSA (2048-bit)
+    BLIND_RSA,
+    // Token Type VOPRF (P-384, SHA-384)
+    VOPRF,
+} as const;
+
+// The Privacy Pass HTTP Authentication Scheme
+//
+// Ref. https://datatracker.ietf.org/doc/draft-ietf-privacypass-auth-scheme/
+//
+// +--------+                               +--------+
+// | Origin |                               | Client |
+// +---+----+                               +---+----+
+//     |                                        |
+//     |      | WWW-Authenticate:         |     |
+//     +----- |    PrivateToken challenge | --->|
+//     |                                        |
+//     |                            (Run issuance protocol)
+//     |                                        |
+//     |      | Authorization:            |     |
+//     |<---- |    PrivateToken token     | ----+
+//     |                                        |
```

Comment:
```suggestion
//   +--------+                              +--------+
//   | Origin |                              | Client |
//   +---+----+                              +---+----+
//       |                                       |
//       +-- WWW-Authenticate: TokenChallenge -->|
//       |                                       |
//       |                            (Run issuance protocol)
//       |                                       |
//       |<------ Authorization: Token ----------+
//       |                                       |
```

draft 14 of Privacy Pass Auth does not present the above diagram, but the suggested one